src/main/java/com/google/devtools/build/lib/skyframe/ConfiguredTargetFunction.java

```diff
@@ -795,33 +791,23 @@ public SkyValue compute(SkyKey key, Environment env) throws ConfiguredTargetFunc
       throw e;
     }
 
-    Map<Label, ConfigMatchingProvider> configConditions = new LinkedHashMap<>();
+    Map<Label, ConfiguredTargetAndData> asConfiguredTargets = new LinkedHashMap<>();
+    Map<Label, ConfigMatchingProvider> asConfigConditions = new LinkedHashMap<>();
```

Comment:
Why use a LinkedHashMap here, and then turn it into an ImmutableMap later on?

Reply:
Good call. Changed to ImmutableMap.Builder.